tests/views_integration/Tribe/Events/Views/V2/Views/Month_ViewTest.php

```diff
@@ -149,10 +149,14 @@ function ( array $data ) use ( $remapped_post_ids ) {
 	}
 
 	/**
-	 * Test render with events with endtime hidden.
+	 * Test render with events with end time hidden.
 	 */
 	public function test_render_with_events_hide_endtime() {
+		// The views will default to `list` because of a dirty context check, so use list here.
+		// It's ok because we are checking the View objects directly. When the dirty context is fixed,
+		// this should change.
```

Comment:
```suggestion
		/* The views will default to `list` because of a dirty context check, so use list here.
		 It's ok because we are checking the View objects directly. When the dirty context is fixed, this should change. */
```
Avoid multi-line `//` comments

Also - is this a stealth TBD to fix the context?

Reply:
Somewhat yes. It's if we decide to refactor View this likely will need updating, as this is testing "Month" but saying it is "list"